tests/deployable.ic-repl.sh

```diff
@@ -67,26 +91,39 @@ function classes_redirect(wasm) {
 
 let S = motoko(file("ok/motoko-instrument.wasm"));
 call S.__get_cycles();
-assert _ == (7199 : int64);
+assert _ == (9003 : int64);
```

Comment:
Check this?

Reply:
We didn't change the instrument module, why does the instruction counter change?

Reply:
It seems to be because I modified the test cases. Namely, `motoko()` calls a few more operations. When I remove those operations, the instruction count reverts to what it was before.